zipline/lib/adjusted_array.py

```diff
@@ -243,6 +243,20 @@ def inspect(self):
         )
 
 
+def ensure_adjusted_array(ndarray_or_adjusted_array, missing_value):
+    if isinstance(ndarray_or_adjusted_array, AdjustedArray):
+        return ndarray_or_adjusted_array
+    elif isinstance(ndarray_or_adjusted_array, ndarray):
+        return AdjustedArray(
+            ndarray_or_adjusted_array, None, {}, missing_value,
```

Comment:
You might want to pass `AdjustedArray.NOMASK` (which is an alias of `None`) for clarity here.